ksqldb-rocksdb-config-setter/src/main/java/io/confluent/ksql/rocksdb/KsqlBoundedMemoryRocksDBConfig.java

```diff
@@ -54,7 +56,18 @@ public class KsqlBoundedMemoryRocksDBConfig extends AbstractConfig {
   private static final double INDEX_FILTER_BLOCK_RATIO_DEFAULT = 0.0;
   private static final String INDEX_FILTER_BLOCK_RATIO_DOC =
       "Percentage of the RocksDB block cache to set aside for high-priority entries, i.e., "
-      + "index and filter blocks.";
+          + "index and filter blocks.";
+
+  public static final String COMPACTION_STYLE_CONFIG = CONFIG_PREFIX + "compaction.style";
+  private static final String COMPACTION_STYLE_DOC =
+      "All RocksDB instances across all KSQL servers will apply this compaction style "
```

Comment:
nit: 
```suggestion
      "All RocksDB instances on this KSQL server will apply this compaction style "
```
The config setter must be set separately for each server instance, so it doesn't make sense to say that setting a config in one place will automatically affect all other servers as well (it will not).

Same for the compression type config below as well.